features/bevragen/persoon/partner/dev/autorisatie-gba.feature

```diff
@@ -0,0 +1,150 @@
+# language: nl
+
+Functionaliteit: autorisatie partners
+
+    Achtergrond:
+      Gegeven de persoon met burgerservicenummer '000000024' heeft een 'partner' met de volgende gegevens
+      | naam                                                                | waarde    |
+      | burgerservicenummer (01.20)                                         | 000000188 |
+      | voornamen (02.10)                                                   | Pieter    |
+      | adellijke titel of predicaat (02.20)                                | JH        |
+      | voorvoegsel (02.30)                                                 | van den   |
+      | geslachtsnaam (02.40)                                               | Aedel     |
+      | geboortedatum (03.10)                                               | 19580701  |
+      | geboorteplaats (03.20)                                              | Berlijn   |
+      | geboorteland (03.30)                                                | 6029      |
+      | geslachtsaanduiding (04.10)                                         | M         |
+      | datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10)  | 20010808  |
+      | plaats huwelijkssluiting/aangaan geregistreerd partnerschap (06.20) | Parijs    |
+      | land huwelijkssluiting/aangaan geregistreerd partnerschap (06.30)   | 5002      |
+      | soort verbintenis (15.10)                                           | P         |
+      En de 'partner' is gewijzigd naar de volgende gegevens
+      | naam                                                         | waarde    |
+      | burgerservicenummer (01.20)                                  | 000000188 |
+      | voornamen (02.10)                                            | Pieter    |
+      | adellijke titel of predicaat (02.20)                         | JH        |
+      | voorvoegsel (02.30)                                          | van den   |
+      | geslachtsnaam (02.40)                                        | Aedel     |
+      | geboortedatum (03.10)                                        | 19580701  |
+      | geboorteplaats (03.20)                                       | Berlijn   |
+      | geboorteland (03.30)                                         | 6029      |
+      | geslachtsaanduiding (04.10)                                  | M         |
+      | datum ontbinding huwelijk/geregistreerd partnerschap (07.10) | 20180317  |
+      | soort verbintenis (15.10)                                    | P         |
+
+
+  Rule: Wanneer met fields gevraagd wordt om een veld waarvoor de gebruiker niet geautoriseerd is, wordt een foutmelding gegeven
+
+    Scenario: Afnemer vraagt om partners en heeft de daarvoor minimale autorisatie
```

Comment:
wat is een "minimale autorisatie"?

Reply:
Afnemer vraagt om partners en heeft uitsluitend de autorisatie die nodig is om deze vraag te mogen stellen